src/core.js

```diff
@@ -917,6 +917,15 @@ $.extend( $.validator, {
 				valid: true,
 				message: this.defaultMessage( element, "remote" )
 			});
+		},
+
+		// cleans up all forms and elements, removes validator-specific events
+		destroy: function() {
+			this.resetForm();
+
+			$( this.currentForm )
+				.off( "submit.validate focusin.validate focusout.validate keyup.validate click.validate" )
```

Comment:
Aah, sorry! This was a nice little feature that I didn't know about until today. :| I'll patch and push.